README.md

```diff
@@ -21,7 +21,9 @@ python3 scripts/train_coarse.py --loop.noise_distribution log_uniform --loop.sig
 
 ### Inference
 
-See `scripts/inference_coarse.py`.
+AMIP SST data must be provided as a conditioning input to inference the coarse models. First run `python3 scripts/download_amip_sst.py` to download the data, and then configure `AMIP_MID_MONTH_SST` and `AMIP_MID_MONTH_SST_PROFILE` in `src/cbottle/config/environment.py` to point to the downloaded data.
```

Comment:
Let's suggest that users use env vars instead
```
export AMIP_MID_MONTH_SST=""
```
We shouldn't encourage editing environment.py directly since this will make it harder for people to update their cod.e